core/pom.xml

```diff
@@ -29,6 +29,7 @@ THE SOFTWARE.
   <parent>
     <groupId>org.jenkins-ci.main</groupId>
     <artifactId>jenkins-parent</artifactId>
+    <!--suppress MavenPropertyInParent -->
```

Comment:
hm?

Reply:
I was getting an error from maven (probably due to my maven version), let me remove it.